.devcontainer/Dockerfile

```diff
@@ -1,47 +1,25 @@
-FROM mcr.microsoft.com/vscode/devcontainers/base:debian-10
+FROM mcr.microsoft.com/vscode/devcontainers/base:debian
 
 WORKDIR /workspaces
 
-# Default ENV
-ENV LANG C.UTF-8
-ENV DEBIAN_FRONTEND noninteractive
-
-# Set shell
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Install docker
-# https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        apt-transport-https \
-        ca-certificates \
-        curl \
-        software-properties-common \
-        gnupg \
-    && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
-    && add-apt-repository "deb https://download.docker.com/linux/debian $(lsb_release -cs) stable" \
-    && apt-get update && apt-get install -y --no-install-recommends \
-        docker-ce \
-        docker-ce-cli \
-        containerd.io \
-    && rm -rf /var/lib/apt/lists/*
+# Set Docker daemon config
+RUN \
+    mkdir -p /etc/docker \
+    && echo '{"storage-driver": "vfs"}' > /etc/docker/daemon.json
 
-# Install tools
-ARG SHELLCHECK_VERSION=0.7.0
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        jq \
+# Installa aditional tools
+RUN \
+    apt-get update \
+    && apt-get install -y --no-install-recommends \
         dbus \
         network-manager \
         libpulse0 \
-        git \
-        tar \
-        xz-utils \
-    && rm -rf /var/lib/apt/lists/* \
-    \
-    && curl -SL "https://storage.googleapis.com/shellcheck/shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz" | tar xJf - \
-    && cp "shellcheck-v${SHELLCHECK_VERSION}/shellcheck" /usr/bin/ \
-    && rm -rf shellcheck-v${SHELLCHECK_VERSION}
+        shellcheck
```

Comment:
How old is the shellcheck?

Reply:
Not that bad, it's from march (0.5.0)

Reply:
Do we not want to be close as possible to CI?

Reply:
Alternatively, there are statically linked versions available on their github releases page...

<https://github.com/koalaman/shellcheck/releases>

Reply:
Yeah, if not using the one in apt, we should use 'stable'